pkg/dynamicmetric/dynamic_spread.go

```diff
@@ -0,0 +1,247 @@
+package dynamicmetric
```

Comment:
move this to the risk package too

Reply:
can't do that. there will be an import cycle

Reply:
pkg/bbgo -> pkg/risk - pkg/bbgo

dynamic_quantity needs pkg/bbgo